app/_styles/scss/Footer.module.scss

```diff
@@ -13,57 +13,159 @@
   padding: 0 42px 0 0;
   padding-bottom: 86px;
 
+  @media (max-width: 600px) {
+    flex-direction: column;
+    padding: 0;
+    padding-bottom: 40px;
+  }
+
   .footerLogoWrapper {
     margin-left: 38px;
     width: 50%;
 
+    @media (max-width: 600px) {
+      margin: 0;
+      width: 100%;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      flex-direction: column;
+    }
+
     .headerLogo {
       display: flex;
+      justify-content: center;
       align-items: center;
       margin-bottom: 58px;
       margin-top: 74px;
+
+      @media (max-width: 600px) {
+        margin: 10px 0 40px 0;
+      }
     }
 
     .headerLogoText {
       font-size: 40px;
       font-weight: 800;
       padding-left: 15px;
+
+      @media (max-width: 600px) {
+        font-size: 20px;
+      }
     }
 
     .textLogo {
       font-size: 16px;
       font-weight: 400;
       color: #797979;
+
+      @media (max-width: 600px) {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        font-size: 12px;
+        margin: 10px;
+      }
     }
   }
 
   .footer_navigation_container {
     width: 50%;
 
+    @media (max-width: 1100px) {
+      gap: 15px;
+    }
+
+    @media (max-width: 600px) {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+      width: 100%;
+      margin-bottom: 20px;
+    }
+
     .navigation {
       display: flex;
       margin-top: 90px;
       justify-content: space-between;
 
+      @media (max-width: 600px) {
+        flex-direction: column;
+        margin-top: 30px;
+      }
+
+      .navigation_title_non {
```

Comment:
please use one styles for classNames 
or snake_case , or camelCase 
please don't mix up and fix in all scss files and tsx files